### ISSUE TITLE
Task serviceLoaderMain fails: Cannot identify ComponentIdentifier subtype. Was OpaqueComponentArtifactIdentifier

### ISSUE THREAD

**Issue by gotson** [autonomousapps/dependency-analysis-android-gradle-plugin#226]
**Build scan link**

**Plugin version**

0.53.1

**Gradle version**

6.3

**Android Gradle Plugin (AGP) version**

**Describe the bug**

The task `buildHealth` fails with the following error:
```
Execution failed for task ':komga:serviceLoaderMain'.
> Cannot identify ComponentIdentifier subtype. Was OpaqueComponentArtifactIdentifier, named webp-imageio-decoder-plugin-0.2.jar
```

**To Reproduce**
1. Clone https://github.com/gotson/komga
2. Check out the branch `dep-analysis`
3. run the `buildHealth` task

**Expected behavior**

The task should run properly.

**Additional context**

The dependency in the error is provided by a jar directly, not from Maven:
```kotlin
  implementation(files("$projectDir/libs/webp-imageio-decoder-plugin-0.2.jar"))
```

**Comment by autonomousapps** [comment]
Thanks for the report.

**Comment by autonomousapps** [comment]
Will be resolved by 0.54.0.
